com.unity.render-pipelines.high-definition/Runtime/RenderPipeline/HDProfileId.cs

```diff
@@ -18,7 +17,8 @@ internal enum HDProfileId
         Distortion,
         AccumulateDistortion,
         ApplyDistortion,
-        DepthPrepass,
+        ForwardDepthPrepass,
```

Comment:
we should rename to ForwardMaterialDepthPrepass and DeferredMaterialDepthPrepass  guess it will be more understandable

Reply:
ah nevermind, I was confuse, keep it like that